function_builder.go

```diff
@@ -1,33 +0,0 @@
-package function
```

Comment:
I don't like the removal of this.

Reply:
This still exists, it was moved along with the constants.  I will extract into a separate PR